src/sync/recoil/onListen.js

```diff
@@ -30,10 +31,10 @@ export default function onListen({ updateItem }) {
   );
 
   listeners.add(
-    'entities__onChange', () => store.entities.onChange(
+    'entities__onChange', () => store.illustrations.onChange(
       async (key, value) => {
         // Update the master entities list
-        const entities = await store.entities.get();
+        const entities = await store.illustrations.get();
         const entitiesAsList = entities.map(([entitiyKey]) => entitiyKey);
         updateItem('entities', entitiesAsList);
```

Comment:
did we want to update the variable names to `illustrations` here as well? Lines 37-39.